ocs_ci/ocs/resources/ocs.py

```diff
@@ -349,3 +352,32 @@ def ocs_install_verification(timeout=600, skip_osd_distribution_check=False):
     assert {defaults.CEPHFS_PROVISIONER, defaults.RBD_PROVISIONER} == (
         {item['metadata']['name'] for item in csi_driver.get()['items']}
     )
+
+    # Verify ceph osd tree have device set PVC names specified
+    log.info("Checking for device set PVC names in ceph osd tree output.")
+    deviceset_pvcs = [
+        item['metadata']['name'] for item in get_all_pvcs(
+            namespace=namespace
+        )['items'] if item['metadata']['name'].startswith(
+            constants.DEFAULT_DEVICESET_PVC_NAME
+        )
+    ]
+    ct_pod = get_ceph_tools_pod()
+    osd_tree = ct_pod.exec_ceph_cmd(ceph_cmd='ceph osd tree', format='')
+    for pvc in deviceset_pvcs:
+        assert f'host {pvc}' in osd_tree, (
+            f"ceph osd tree output does not contain pvc name {pvc}"
+        )
+    log.info("Verified device set PVC names in ceph osd tree output.")
+
+    # Verify OSDs are listed as sdd
+    log.info("Verify OSDs are listed as sdd in ceph osd tree output.")
+    for num in range(len(deviceset_pvcs)):
+        re_exp = f'ssd [0-9]*.[0-9]*[ ]*osd.{num}'
```

Comment:
this is dependent on hardware we run and can easily fail if we move out of aws or different systems?

Reply:
This part is removed now. It will be added in another PR after getting this clarity.